testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -137,6 +137,7 @@
     "LAMMPShyd", "LAMMPShyd2",
     "LAMMPSdata_deletedatoms",  # with deleted atoms
     "LAMMPSDUMP",
+    "LAMMPSDUMP_long",  # lammpsdump file with extra columns
```

Comment:
might be worth defining what extra columns somewhere

Reply:
I might generate some new test data to cover a few of the other coordinate conventions now that you mention it.